buddyboss/buddyboss-platform
# PROD-726 Fix : #3031 Fixed photo upload at admin side.

### Changes proposed in this Pull Request:



Fixes #3031.

### How to test the changes in this Pull Request:

1. Go to the admin user section
2. Click on edit go extended profile
3. Scroll down to and find the profile photo and try to upload a new one.
4. You can see it working.

### Proof Screenshots or Video


https://prnt.sc/1w0yhh5
